doc/source/management/super_read_only.rst

```diff
@@ -4,7 +4,7 @@
  ``super-read-only`` option 
 ============================
 
-|Percona Server| has ported a new option, :variable:`super_read_only` from *WebScaleSQL*, which is like the `read-only <http://dev.mysql.com/doc/refman/5.6/en/server-system-variables.html#sysvar_read_only>`_ option, but affecting users with ``SUPER`` privileges as well. Enabling :variable:`super_read_only` implies regular ``read-only`` as well.
+|Percona Server| has ported a new option, :variable:`super_read_only` from *WebScaleSQL*, which is like the `read-only <http://dev.mysql.com/doc/refman/5.6/en/server-system-variables.html#sysvar_read_only>`_ option, but affecting users with ``SUPER`` privileges as well. This means that server will not allow any updates, beside updating the replication repositories (``CHANGE MASTER``, ``START SLAVE`` and ``STOP SLAVE``), even for the users that have ``SUPER`` privilege. Replication repositories can be updated only if slave status logs are tables, not files: when :variable:`master-info-repository` is set to ``TABLE`` and/or :variable:`relay-log-info-repository` is set to ``TABLE``. Enabling :variable:`super_read_only` implies regular ``read-only`` as well.
```

Comment:
Needs rephrasing, because replication repositories can be updated in any case, if they are FILEs, then SRO does not apply to them at all.